### PR TITLE
[Do Not Merge] NetStandard Upgrade: Update NuSpec & Raw NuGets

### PR DESCRIPTION
**Pull request by jamesmontemagno**
Should now install in just about every application

### CODE REVIEW

**Comment by jamesmontemagno** [comment]
@onovotny can you validate my changes here.

File: sqlite-net-pcl.nuspec [hunk 1]
```diff
@@ -1,39 +1,61 @@
...
+        </group>
+        <group targetFramework="wp">
+            <dependency id="SQLitePCLRaw.bundle_green" version="1.1.5" />
+        </group>
+        <group targetFramework="wpa">
+            <dependency id="SQLitePCLRaw.bundle_green" version="1.1.5" />
+        </group>
+        <group targetFramework="netstandard1.1">
+            <dependency id="SQLitePCLRaw.bundle_green" version="1.1.5" />
+            <dependency id="NETStandard.Library" version="1.6.0" />
```
Comment (onovotny) [inline sqlite-net-pcl.nuspec]: NETStandard.Library 1.6.1 is better since it has fixes for Xamarin related issues with certain PCL profiles

**Comment by onovotny** [comment]
Also, do you want dependency groups for Xamarin tvOS and watchOS?

**Comment by jamesmontemagno** [comment]
I think this library itself is still actually a PCL which is really odd, I am going to change it over to .NET Standard 1.1 and then add tvos/watchos

File: sqlite-net-pcl.nuspec (hunk 1 above)
Comment (jamesmontemagno) [inline sqlite-net-pcl.nuspec]: sounds good :) yeah I see that is the default now of file new

**Comment by jamesmontemagno** [comment]
Alright @onovotny I think this should be good now, I have created a new .NET Standard 1.1 project and added in the references needed.

**Comment by jamesmontemagno** [comment]
I think that once Bitrise upgrades to the latest stable it will build up.

**Comment by onovotny** [comment]
@jamesmontemagno if you're going that approach, you can just use the pack targets too -- `dotnet pack`. All of the info in the nuspec have equivalent properties in the csproj. The only reason to really split out the dependency groups as you have them would be to potentially minimize pain on people using `packages.config` with older project types.

**Comment by praeclarum** [comment]
I have been working on a new version of this that builds using .NET Core on Bitrise.

I'm making good progress! Let's switch to there.

https://github.com/praeclarum/sqlite-net/pull/561

**Pull request closed without merging**